src/azure-cli/azure/cli/command_modules/mysql/_util.py

```diff
@@ -374,19 +373,12 @@ def _resolve_api_version(client, provider_namespace, resource_type, parent_path)
 
 def run_subprocess(command, stdout_show=None):
     if stdout_show:
-        process = subprocess.Popen(command, shell=True)
+        process = run_cmd(command)
     else:
-        process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        process = run_cmd(command, capture_output=True)
     process.wait()
     if process.returncode:
-        logger.warning(process.stderr.read().strip().decode('UTF-8'))
-
-
-def run_subprocess_get_output(command):
-    commands = command.split()
-    process = subprocess.Popen(commands, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    process.wait()
-    return process
+        logger.warning(process.stderr.strip().decode('UTF-8'))
```

Comment:
```suggestion
        if stdout_show:
          logger.warning("Error in running cmd {}".format("|".join(command)))
        else:
          logger.warning(process.stderr.strip().decode('UTF-8'))
```

Reply:
When `capture_output=False`, stderr would be printed and `process.stderr` would be `None`